enterprise/frontend/src/embedding-sdk/cli/steps/install-sdk.ts

```diff
@@ -0,0 +1,92 @@
+import { exec as execCallback } from "child_process";
+
+import { detect } from "detect-package-manager";
+import toggle from "inquirer-toggle";
+import ora from "ora";
+import { promisify } from "util";
+
+import { SDK_PACKAGE_NAME } from "embedding-sdk/cli/constants/config";
+import type { CliStepMethod } from "embedding-sdk/cli/types/cli";
+import {
+  printEmptyLines,
+  printInfo,
+  printSuccess,
+} from "embedding-sdk/cli/utils/print";
+
+export const getInstallCommand = async (packageName: string) => {
```

Comment:
This isn't used outside this file.
```suggestion
const getInstallCommand = async (packageName: string) => {
```